CarAccidentErrorHandling/emergencyServices.ts

```diff
@@ -0,0 +1,29 @@
+export function callEmergencyServices() {
+  console.log("Emergency services are being contacted.");
+}
+
+export function callTowTruck() {
+  console.log("Tow truck is on the way to clear the vehicle.");
+}
+
+export function assessInjuries() {
+  console.log(
+    "Injuries are being assessed, and medical assistance is provided if needed."
```

Comment:
I don't think this is too long to send it in the next line. we can have this in the same line itself.